### DISCUSSION TITLE
zsh: command not found: graphclient

### DISCUSSION THREAD

**Discussion by taetaehoho** [graphprotocol/graph-client#267] [Q&A] [answered]
Hey! 

I ran `yarn add --dev @graphclient/client-cli` but when I run graphclient build (with a .graphql and a .graphclientrc in the same dir) it continues to give me "zsh: command not found: graphclient". 

I've also tried installation with npm. Wondering what i'm doing wrong!

**Comment by dotansimha** [comment, marked as answer]
Hi there! if you are using Yarn, you should run `yarn graphclient`, it has a built-in mechanism for resolving local `bin`s.
